src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2039,16 +2039,27 @@ jint os::init_2(void) {
     if (status != 0) {
       log_info(os)("os::init_2 getrlimit failed: %s", os::strerror(errno));
     } else {
+      rlim_t rlim_org = nbr_files.rlim_cur;
```

Comment:
Nit: can you use `rlim_orig` instead of `rlim_org` please.

Reply:
I always struggle with how to shorten variable names. I guess when I need to shorten a word, I try to stay at 3 letters (original DOS extension limit?)

What if I simply use "rlim_original"?

It's only used one time and the code line where it's used is short enough to fit it.